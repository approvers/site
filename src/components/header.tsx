import Link from "next/link";
import styles from "../scss/components/header.module.scss";

export const Header = (): JSX.Element => (
  <header className={styles.headerLayout}>
    <Link href="/">
      <a className={styles.hero}>
        <img src="/android-chrome-512x512.png" width={512} height={512} alt="Approversロゴ"></img>
        Approvers
      </a>
    </Link>
  </header>
);
