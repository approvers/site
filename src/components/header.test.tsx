import { Header } from "./header";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
