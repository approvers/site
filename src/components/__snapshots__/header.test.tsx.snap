// Vitest Snapshot v1

exports[`renders correctly 1`] = `
<header
  className="css-plc0gq"
>
  <a
    className="css-70qvj9"
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <span
      className="chakra-avatar css-11wutbi"
    >
      <div
        aria-label="Approversロゴ"
        className="chakra-avatar__initials css-0"
        role="img"
      >
        A
      </div>
    </span>
    <span
      className="chakra-heading css-0"
    >
      Approvers
    </span>
  </a>
</header>
`;
